src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import Tooltip from '~shared/Extensions/Tooltip';
+import Icon from '~shared/Icon';
+import { useMembersSubNavigation } from './hooks';
+import styles from './SubNavigation.module.css';
+
+const SubNavigation = () => {
+  const { handleClick, isCopyTriggered } = useMembersSubNavigation();
+  const { formatMessage } = useIntl();
+
+  return (
+    <ul>
+      <li>
+        <button
+          type="button"
+          onClick={handleClick}
+          className={styles.navigationButton}
+        >
+          <Tooltip
+            tooltipContent={
+              <span className="text-3 underline">
+                {formatMessage({
+                  id: isCopyTriggered ? 'copiedColony' : 'copyColony',
+                })}
+              </span>
+            }
+            isSuccess={isCopyTriggered}
+          >
+            <Icon name="share-network" appearance={{ size: 'small' }} />
+            <span className="text-md ml-2">Invite members</span>
```

Comment:
Are these values shouldn't be used with translation?

Reply:
Translations added